dataprofiler/tests/data/csv/empty_rows.txt

```diff
@@ -0,0 +1,9 @@
+NAME, VALUE
```

Comment:
this is more ticky tacky but this file has nulls across a row while the iris one below has nulls randomly which is good but the file is super long.

it would be good to just truncate the iris null rows and it would make it more readable IMO. Is there an intention to having such  a long file?

Reply:
I wanted this file because it the direct file that caused the issue. And I wanted a modified iris (with mix of null rows and null in rows) because it is a much larger and would cover more bases in my opinion. I will truncate iris somewhat.